bsp/nrf5x/libraries/drivers/drv_log.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Not recommend to use a unify header for log. You can define the log tag in each module.